fuse/meson.build

```diff
@@ -0,0 +1,32 @@
+mount_afp_sources = ['client.c']
+
+mount_afp = executable(
+  'mount_afp',
+  sources: mount_afp_sources,
+  include_directories: incdir,
+  link_with: libafpclient,
+  c_args: cflags,
+  install: true,
+)
+
+afpfsd_sources = ['commands.c', 'daemon.c', 'fuse_int.c', 'fuse_error.c']
+
+afpfsd = executable(
+  'afpfsd',
+  sources: afpfsd_sources,
+  include_directories: incdir,
+  link_with: libafpclient,
+  c_args: cflags,
+  dependencies: [fuse_dep],
+  install: true,
+)
+
+meson.add_install_script(
+  'sh',
+  '-c',
+  'ln -sf mount_afp $DESTDIR@0@/afp_client'.format(get_option('prefix') / get_option('bindir'))
+)
```

Comment:
You don't specify a minimum meson version you are targeting.

Since meson 0.61, you don't need this anymore since there's a builtin: https://mesonbuild.com/Reference-manual_functions.html#install_symlink

Reply:
Good call, thanks.